framework/include/systems/LinearSystem.h

```diff
@@ -115,6 +115,12 @@ class LinearSystem : public SolverSystem, public PerfGraphInterface
   virtual TagID systemMatrixTag() const override { return _system_matrix_tag; }
   ///@}
 
+  /// Fetching the right hand side vector from the libmesh system.
```

Comment:
should these be const?
very many objects can get references to the system, so they could get non-const access to rhs & matrix

Reply:
I'll add a const accessor too. For the tagging they can't be const considering things need to contribute to them.